pages/linux/cat.md

```diff
@@ -0,0 +1,32 @@
+# cat
+
+> Print and concatenate files.
+> More information: <https://www.gnu.org/software/coreutils/cat>.
+
+- Print the contents of a file to the standard output:
+
+`cat {{path/to/file}}`
+
+- Concatenate several files into an output file:
+
+`cat {{path/to/file1 path/to/file2 ...}} > {{path/to/output_file}}`
+
+- Append several files to an output file:
+
+`cat {{path/to/file1 path/to/file2 ...}} >> {{path/to/output_file}}`
+
+- Copy the contents of a file into an output file without buffering:
```

Comment:
```suggestion
- Copy the contents of a file into an output file without b[u]ffering:
```

Optional fix: mnemonic can be added.

Reply:
What? AFAIK the `u` stands for "unbuffered".

Reply:
But there is no such word in description, the closest one is `buffering`. I can suggest another description to make a better mnemonic.